climada/hazard/trop_cyclone/__init__.py

```diff
@@ -0,0 +1,5 @@
+from climada.hazard.trop_cyclone.trop_cyclone import *
+from climada.hazard.trop_cyclone.trop_cyclone_windfields import compute_windfields_sparse, compute_angular_windspeeds, tctrack_to_si, \
```

Comment:
Use local imports
```suggestion
from .trop_cyclone import *
from .trop_cyclone_windfields import compute_windfields_sparse, compute_angular_windspeeds, tctrack_to_si, \
```

Reply:
fine by me (I assume this is the pattern for init files)